internal/pkg/workspace/workspace.go

```diff
@@ -117,6 +118,16 @@ func (ws *Workspace) Summary() (*Summary, error) {
 
 // ServiceNames returns the names of the services in the workspace.
 func (ws *Workspace) ServiceNames() ([]string, error) {
```

Comment:
I'm a little worried that in service-heavy (and soon, job-heavy) packages, this will be confusing because there are other service-related (and soon, job-related) methods and vars. ServiceNames() is being called in places like selector, pipeline, storage.... Seems like a method called just "Services" would be returning service objects, no?

Reply:
Sounds good!